src/Data/Integer/GCD.agda

```diff
@@ -44,5 +48,17 @@ gcd[i,j]≡0⇒i≡0 i j eq = ∣i∣≡0⇒i≡0 (ℕ.gcd[m,n]≡0⇒m≡0 (+-i
 gcd[i,j]≡0⇒j≡0 : ∀ {i j} → gcd i j ≡ 0ℤ → j ≡ 0ℤ
 gcd[i,j]≡0⇒j≡0 {i} eq = ∣i∣≡0⇒i≡0 (ℕ.gcd[m,n]≡0⇒n≡0 ∣ i ∣ (+-injective eq))
 
-gcd-comm : ∀ i j → gcd i j ≡ gcd j i
+gcd-comm : Commutative gcd
 gcd-comm i j = cong (+_) (ℕ.gcd-comm ∣ i ∣ ∣ j ∣)
+
```

Comment:
1. This is great,
2. Shouldn't all these things be in `Data.Integer.GCD.Properties`?

Reply:
There is no such `Properties` file. The dependencies for the more "complex" operations are still snarled up. I've been trying to gradually migrate all operations into the `Base` files, leaving e.g. `Data.Integer.GCD` as the de-facto properties file, but I've got stuck on the dependency on the proof that `_<_` is well-founded.

Therefore I don't think @Lysxia should responsible for changing this. I think its fine here at the moment.